### ISSUE TITLE
new update process for the pwa

### ISSUE THREAD

**Issue by JeanBarriere** [storyscript/dashboard#146]
Based from the pwa update

tl;dr -- we need to check the pwa update in a non-blocking way

### Observations

As a user,
I arrive on the dashboard and it takes time before I can access anything (public or private view)
I should be able to arrive there in less time,
So that the experience is smoother and faster.

### Acceptance
**Given** a user access to the dashboard
**When** it hits the initializer view
**Then** we check for pwa in the background but don't block the flow of user retrieval

### Notes

@aydaoz already worked on a first iteration on what displaying update information would look like: 
![image](https://user-images.githubusercontent.com/11390722/65606104-8e849a00-dfaa-11e9-9576-a938f5de907d.png)

_Originally posted by @JeanBarriere in https://github.com/storyscript/dashboard/issues/122#issuecomment-535027548_
